distribution/lib/Standard/Table/0.0.0-dev/src/Internal/Widget_Helpers.enso

```diff
@@ -59,7 +59,7 @@ make_aggregate_column_selector table display=Display.Always include_group_by=Tru
 make_aggregate_column_vector_selector : Table -> Display -> Widget
 make_aggregate_column_vector_selector table display=Display.Always =
     item_editor = make_aggregate_column_selector table display=Display.Always
-    Vector_Editor item_editor=item_editor item_default=item_editor.values.first.value display=display
+    Vector_Editor item_editor=item_editor item_default="(Aggregate_Column.Group_By)" display=display
```

Comment:
Shall we have a #TODO here linking to the ticket for fixing this? IIRC our discussion this is a workaround for some bug where the value inserted is wrong for some reason when adding a new item.